sooyoung/services/userService.js

```diff
@@ -10,9 +12,36 @@ const signUp = async (username, email, profileImage, password) => {
     err.statusCode = 400;
     throw err;
   }
-  return await userDao.createUser(username, email, profileImage, password);
+
+  const COST_FACTOR = 12;
+  const hashedPassword = await bcrypt.hash(password, COST_FACTOR);
+
+  return await userDao.createUser(
```

Comment:
```suggestion
  return userDao.createUser(
```
이후에는 동기적으로 실행할 코드가 없기때문에 await를 빼주시는게 깔끔합니다.